client_interface.py

```diff
@@ -62,7 +72,7 @@ def send_state_to_server(self):
             message["interface_data"]["game_round"] = self.game_state.game_round
             asyncio.ensure_future(self.ws.send_json(message))
 
-    async def get_messages(self, robot_name, own_robot_name):
+    async def get_messages(self, robot_name, own_robot_name=""):
```

Comment:
Tady mám jen dotaz proč je tu own_robot_name="" 
Nejde to rovnou tedy taky zadat jako argument to vlastní jméno při spuštění interface.py?

Reply:
Myslis, ze si toho robota rovnou prejmenuju? To tam predtim nebylo, tak jsem to nechala tak: pokud mu nebude dano jmeno z welcome_boardu, dosadi se to nactene z yamlu. Ma to fungovat jinak?

Reply:
Nene dobrý, jako jo myslela jsem že by se mohlo rovnou i přejmenovat ale je to zbytečné, protože vlastně když spouštíš interface přímo tak netřeba, když si chceš vybírat, pustíš si welcome board. Právě jsem dodělala zadání počtu hráčů na serveru, tak během dneška ještě nahraju :-D